README.md

```diff
@@ -37,24 +37,40 @@ In order to launch a new EC2 instance, customers need to specify a lot of option
 - Config file for more convenient launch
 
 ## Installation and Configuration
+1. Install AWS CLI
 
-1. Install AWS Simple EC2 CLI
+To execute the CLI, you will need AWS credentials configured. Take a look at the [AWS CLI configuration documentation](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html#config-settings-and-precedence) for details on the various ways to configure credentials. Alternatively, you can try out the AWS Simple EC2 CLI by populating following environment variables:
 
 ```
-go install simple-ec2
+export AWS_ACCESS_KEY_ID="..."
+export AWS_SECRET_ACCESS_KEY="..."
+# Set default Region (optional)
+export AWS_REGION="us-east-1" 
 ```
-
 2. Install sshpass (1.06+)
 
-To install sshpass, you can refer to this [guide](https://gist.github.com/arunoda/7790979). Note that sshpass 1.05, which is commonly used, is not compatible with this tool. 
+To install sshpass (1.06+), you can refer to this [guide](https://gist.github.com/arunoda/7790979). Please note install sshpass version higher than 1.06. 
```

Comment:
[nit] "Please note install" sounds off, maybe change to "Please note *to* install..." or "Make sure to install..."

Reply:
good catch! I updated it